pages/en/developing/assemblyscript-api.mdx

```diff
@@ -761,3 +761,14 @@ The base `Entity` class and the child `DataSourceContext` class have helpers to
 - `getBytes(key: string): Bytes`
 - `getBoolean(key: string): boolean`
 - `getBigDecimal(key: string): BigDecimal`
+
+### Common AssemblyScript Issues
+
+There are certain [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) issues that are common to run into during subgraph development. They range in debug difficulty, however, being aware of them may help. The following is a non-exhaustive list of these issues:
+
+- Be careful with handler computation limits, especially when using the `.pow()` function with large numbers.
+  - As of [`graph-node v0.26.0`](https://github.com/graphprotocol/graph-node/releases/tag/v0.26.0) gas metering was introduced.
```

Comment:
I don't think the goal here was to penalise reasonable actions in mappings (cc @leoyvens), can you give more detail on when you encountered an issue with `.pow()` (the gas setting might not be appropriate)

Reply:
There once was a gas issue with `pow` but that was fixed. The issue with abusing `pow` on `BigInt` is that it can cause the number to quickly grow in size.

Reply:
removed